File: medium/pig_latin/pig_latin.kt
fun pig_latin(text:String):String{
    
    var text_list = text.split(" ")
    var text_pig = ""
    for(i in text_list){
        text_pig = text_pig + i.substring(1,i.length) + i.substring(0,1) + "ay "
    }
	 
    return(text_pig)
}	



fun main(args: Array<String>){
	
    println("Text : ")
    var text = readLine()
    
    println(pig_latin(text))     
      
}

File: medium/pig_latin/pig_latin.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoloLearnPrograms
{
    internal class Program
    {

        public static string PigLatin(string text)
        {

            string textConvertido = "";

            string[] words = text.Split(' ');


            foreach (string word in words)
            {
                textConvertido += word.Substring(1, word.Length - 1) + word[0] + "ay " ; 
            }

            return textConvertido;

        }

        static void Main(string[] args)
        {
            
            string text = Console.ReadLine();

            string conversion = PigLatin(text);

            Console.WriteLine(conversion);


            


        }
    }
}
